python/feathub/feature_tables/sinks/redis_sink.py

```diff
@@ -11,39 +11,46 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-from typing import Dict
+from typing import Dict, Union
 
+from feathub.common.exceptions import FeathubException
 from feathub.feature_tables.sinks.sink import Sink
+from feathub.feature_tables.sources.redis_source import RedisMode
 
 
 class RedisSink(Sink):
     def __init__(
         self,
         host: str,
         port: int = 6379,
+        mode: Union[RedisMode, str] = RedisMode.STANDALONE,
         username: str = None,
         password: str = None,
         db_num: int = 0,
         namespace: str = "default",
     ):
         """
+        :param mode: The deployment mode or the name of the mode of the redis service.
```

Comment:
Can you put the params doc in the same order as the parameters in the signature?